fossevents/templates/base.html

```diff
@@ -40,10 +65,8 @@
 
     </main>
 
-    <footer class='site-footer'>
```

Comment:
Please do not change the semantics of page, and checkout the existing rules applied before making any change. The footer was made in a way that it sticks to footer always, even if there no content on page: https://github.com/Kushagra343/fossevents.in/blob/user-interface/fossevents/static/css/_layout.scss#L1-L25

This global rule is applied to other pages like about.html, privacy.html.